storage/ctab/src/main/java/org/openscience/cdk/io/RdfileReader.java

```diff
@@ -0,0 +1,533 @@
+package org.openscience.cdk.io;
+
+import org.openscience.cdk.exception.CDKException;
+import org.openscience.cdk.interfaces.IChemObject;
+import org.openscience.cdk.interfaces.IChemObjectBuilder;
+import org.openscience.cdk.tools.ILoggingTool;
+import org.openscience.cdk.tools.LoggingToolFactory;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Iterating reader for RDFiles.
```

Comment:
Please elaborate, e.g. by pointing to the specification.

Reply:
Yes, I will.
I wanted to get some basic confirmation that you are happy with the general architecture, functionality before starting to write documentation.